com.unity.formats.fbx/package.json

```diff
@@ -16,4 +16,4 @@
     "maya",
     "max"
   ]
-}
+}
```

Comment:
can you drop this change? different text editors fight over whether there's a newline at the end

Reply:
discussed offline to leave this as it comes from a commit from the merge